src/ray/raylet/scheduling/cluster_resource_manager.h

```diff
@@ -109,15 +109,6 @@ class ClusterResourceManager {
   bool AddNodeAvailableResources(scheduling::NodeID node_id,
                                  const ResourceSet &resource_set);
 
-  /// Update node available resources.
-  /// NOTE: This method only updates the existing resources of the node, and the
```

Comment:
IIUC, this means gcs won't see dynamic resources (e.g. PG resources) created by raylets. Although I think it's no harm to let gcs see those resources as well.